docs/pages/eas/workflows/syntax.mdx

```diff
@@ -180,6 +180,90 @@ on:
     - cron: '0 0 * * *' # Runs at midnight GMT every day
 ```
 
+### `on.workflow_dispatch.inputs`
+
+Defines inputs that can be provided when manually triggering a workflow using the `eas workflow:run` command. This allows you to create parameterized workflows that can accept different values each time they run.
+
+```yaml
+on:
+  workflow_dispatch:
+    # @info #
+    inputs:
+      name:
+        type: string
+        required: false
+        description: 'Name of the person to greet'
+        default: 'World'
+      choice_example:
+        type: choice
+        options:
+          - to be
+          - not to be
+        required: true
+    # @end #
+```
+
+#### Input Properties
+
+Each input can have the following properties:
```

Comment:
```suggestion
```

No other section on this page has this section when describing properties table.